spec/models/spree/product_spec.rb

```diff
@@ -309,7 +309,7 @@ module Spree
       it "defaults available_on to now" do
         Timecop.freeze do
           product = Product.new
-          expect(product.available_on).to eq(Time.zone.now)
+          expect(product.available_on).to be_within(0.000001).of(Time.zone.now)
```

Comment:
I'm just wondering, would `be_within(1)` be good enough?

Reply:
Hm, I'm using the full precision of the timestamp but you are probably right, from a user perspective, "now" just needs the accuracy to the second.